Foodforce2.py

```diff
@@ -55,7 +55,7 @@
 
 select_lang_flag = 1
 desktop2 = gui.Desktop()
-set_icon(pygame.image.load(os.path.join('data', 'WFPLOGO.png')).convert_alpha())
+# set_icon(pygame.image.load(os.path.join('data', 'WFPLOGO.png')).convert_alpha())
```

Comment:
If the line doesn't serve any purpose, could you delete instead of commenting it out?